src/featureParameters/IsEnabled_BDICMT.featureParameterBoolean

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FeatureParameterBoolean xmlns="http://soap.sforce.com/2006/04/metadata">
+    <dataflowDirection>SubscriberToLmo</dataflowDirection>
+    <masterLabel>Is Custom Metadata Mapping Enabled</masterLabel>
```

Comment:
@psadie Since this is unique to BDI, can you update the label to something like "Is BDI Custom Metadata Type mapping enabled"?

Reply:
@force2b - absolutely, new commit is up 👍